docs/src/gui/qtvcp.adoc

```diff
@@ -267,197 +387,276 @@ net s32-in1        test_panel.doublescale_1-s   classicladder.0.s32in-00
 start
 ----
 
-In this case we load qtvcp using -Wn; which waits for the panel to finish loading before
-continuing to run the next HAL command. This is so the HAL pins from the panel are finished
-in case the are used in the rest of the file.
+<1> In this case we load `qtvcp` using *`-Wn`* which waits for the panel
```

Comment:
Is <1> needed in the code?
And I guess "<1>" should be a footnote referring to "<1>" ?
Unfortunately "<1>" is formatted as "1."

![grafik](https://user-images.githubusercontent.com/67957916/189978014-c2ea11c6-c4ba-4d0a-bf8d-0b93082e3b60.png)

So not really clear.

Reply:
Yes "<1>" must be in the code block. See https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/

About formatting, maybe this can be customized to our likings, but this is quite common rendering.

Reply:
I wouldn't associate the <1> with the list item "1."
In the linked examples it is formatted both as  ① which makes it clear.
And they added the <1> as a comment in the code in the second example.
So if this isn't supported by asciidoc-py, we should format it in another way to make it clear.
First have equal symbols and second maybe add the explanation as a footnote.

Reply:
I fixed the missing comment character.

From https://asciidoc-py.github.io/userguide.html#X105 it seems we're missing the `-a icons` command-line option for fine rendering.

Reply:
>From https://asciidoc-py.github.io/userguide.html#X105 it seems we're missing the -a icons command-line option for fine rendering.

Great! That seems to resolve this issue.

Reply:
I just need to be pointed to the right place if we want this option to be added with this PR or a later (dedicated ?) one... Please tell me.

Reply:
I'll check that and commit it separately.

Reply:
Looks like a bigger change. `-a icons` tells asciidoc to use icons globally, so for icons like [NOTE], [EXAMPLE] etc. as well.
And as we don't have them, it complains about it. And we have to add icons for (1), (2) etc. or adapt the preprocessor to use the unicode characters for that.

Reply:
Yeah, but than we have to copy them somewhere and make sure that they are on the right place for all languages and so on.